src/Z3/Base.hs

```diff
@@ -1128,7 +1129,9 @@ mkMod = liftFun2 z3_mk_mod
 mkRem :: Context -> AST -> AST -> IO AST
 mkRem = liftFun2 z3_mk_rem
 
--- TODO: Z3_mk_power
+-- | Create an AST node representing arg1 ^ arg2.
+mkPow :: Context -> AST -> AST -> IO AST
```

Comment:
I think it's better to give it the same name as in the Z3 API (but camel case), that is `mkPower`.

Reply:
Fair point :+1: